src/shared/components/ErrorBoundary.tsx

```diff
@@ -1,5 +1,6 @@
 // Libraries
 import React, {Component, ErrorInfo} from 'react'
+import _ from 'lodash'
```

Comment:
be more specific on the imports, like `import {isEqual} from 'lodash'` instead. every couple months we get a "LETS REMOVE ALL OF THE LODASH" bug under the skin, and having the explicit mappings makes it easier to whoever takes it on

Reply:
good call!